sros2/sros2/api/_utilities.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2020 Canonical Ltd
```

Comment:
This should retain the original copyright notice. The code in this file predates 2020 and was not written solely by Canonical.

Similarly history should be preserved

Reply:
Good catch, thank you. I also redid this to keep history, but I suspect that means this cannot be squashed.